pkg/services/accesscontrol/api/api.go

```diff
@@ -101,7 +115,7 @@ func (api *AccessControlAPI) searchUserPermissions(c *contextmodel.ReqContext) r
 	userIDString := web.Params(c.Req)[":userID"]
 	userID, err := strconv.ParseInt(userIDString, 10, 64)
 	if err != nil {
-		response.Error(http.StatusBadRequest, "user ID is invalid", err)
+		return response.Error(http.StatusBadRequest, "user ID is invalid", err)
```

Comment:
:flushed:

Reply:
Yeah :sweat_smile: 
I just noticed there is the same a few lines below. But I'll tackle this in another PR (by removing the endpoint altogether :grimacing: )

Reply:
Oh nooo 🙈